res/css/views/settings/tabs/user/_GeneralUserSettingsTab.scss

```diff
@@ -15,7 +15,8 @@ limitations under the License.
 */
 
 .mx_GeneralUserSettingsTab_changePassword .mx_Field,
-.mx_GeneralUserSettingsTab_themeSection .mx_Field {
+.mx_StyleUserSettingsTab_themeSection .mx_Field,
+.mx_StyleUserSettingsTab_fontScaling .mx_Field {
```

Comment:
These couple `StyleUserSettingsTab` rules should be in their own file, though I think it got renamed.